src/utils/fees.js

```diff
@@ -12,7 +12,8 @@ const SEND_FEE_UNITS = {
   SOL: 1000000,
   MATIC: 21000,
   ERC20: 90000,
-  ARBETH: 620000
+  ARBETH: 620000,
+  FUSE: 0000000001
```

Comment:
Does this mean just `1`? I'm surprised that the gas in fuse is not same as eth for sending (21000)

Reply:
ref: https://explorer.fuse.io/tx/0xbb1cdab2f547b649eaca6ccc7b241bd643b89786602fa75b62306ca3007c197c/logs

<img width="458" alt="Screenshot 2021-09-07 at 14 04 00" src="https://user-images.githubusercontent.com/57037080/132383745-e1d7187f-d2df-4d3f-8c1e-414124f076b1.png">

Reply:
1 Gwei is the gas price, the gas limit in this screenshot is `91,366` This is what is meant by "fee units" in our case.
And the tx you have presented does not look like a regular TX, but a smart contract call. "SEND_FEE_UNITS" is for the most basic, sending Fuse from one address to another, i think it's probably 21000 like the other ethereum chains but you should try it and see
Gas price is always retrieved from the nodes using eth_gasPrice or another api

Reply:
Thanks I will use ```eth_gasPrice```